style.css

```
@@ -976,34 +976,43 @@ body.track-size-s .clip.selected {
-  box-sizing: border-box;
+  box-shadow: 0 0 0 9999px rgba(0, 0, 0, 0.6);
+  border: 1px solid rgba(255, 255, 255, 0.88);
   pointer-events: none;
 }
+#exportFrameOverlay.is-crop .ef-shade {
+  box-shadow: 0 0 0 9999px #000;
+}
+#btnExportFrameDim .ico { margin-right: 0; }
+#btnExportFrameDim .ef-bulb-rays { opacity: 0; }
+#btnExportFrameDim.on .ef-bulb-rays { opacity: 1; }
+#btnExportFrameDim.on .ef-bulb-glass { fill: currentColor; fill-opacity: .22; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)